rules.py

```diff
@@ -953,25 +953,30 @@ def check(self, element):
         object_id = element.get("objectId")
         if element.tag == "Candidate":
             ballot_name = element.find("BallotName")
-            if ballot_name.find("Text") is not None:
-                name = ballot_name.find("Text").text
```

Comment:
You can combine everything in one line here - name = element.find("BallotName").find("Text").text

Reply:
Updated the conditional checks and covered the corner cases.